tests/lit-tests/debug_type_dwarf.ispc

```diff
@@ -0,0 +1,12 @@
+// RUN: %{ispc} --target-os=windows --target=host --nostdlib --nowrap --emit-llvm-text -o - %s 2>&1 | FileCheck %s --implicit-check-not "llvm.dbg.cu"
+// RUN: %{ispc} -g --target-os=windows --target=host --nostdlib --nowrap --emit-llvm-text -o - %s 2>&1 | FileCheck %s -check-prefix=CHECK_CODEVIEW
+// RUN: %{ispc} -g --target=host --nostdlib --nowrap --emit-llvm-text -o - %s 2>&1 | FileCheck %s -check-prefix=CHECK_CODEVIEW
+// RUN: %{ispc} --dwarf-version=3 --target-os=windows --target=host --nostdlib --nowrap --emit-llvm-text -o - %s 2>&1 | FileCheck %s -check-prefix=CHECK_DWARF
+// RUN: %{ispc} -g --dwarf-version=3 --target-os=windows --target=host --nostdlib --nowrap --emit-llvm-text -o - %s 2>&1 | FileCheck %s -check-prefix=CHECK_DWARF
+// RUN: %{ispc} --dwarf-version=3 -g --target-os=windows --target=host --nostdlib --nowrap --emit-llvm-text -o - %s 2>&1 | FileCheck %s -check-prefix=CHECK_DWARF
```

Comment:
The checks on line 5 and 6 are the same with different order of `-g`/`-dwarf-version` options. Is it intended?

Reply:
Yes, it is intended to check that `-g` logic doesn't redefine debug info type.